docs/Client.md

```diff
@@ -99,19 +87,15 @@ const client = Client('http://localhost:3000')
 
 ## Instance Methods
 
-### `Client.close()` _(2 overloads)_
+### `Client.close([ callback ])` 
 
-Closes the client and gracefully waits for enqueued requests to complete before invoking the callback (or returnning a promise if no callback is provided).
-
-#### (1) `Client.close()`
-
-Returns: `Promise<null>`
-
-#### (2) `Client.close(callback)`
+Closes the client and gracefully waits for enqueued requests to complete before resolving.
 
 Arguments:
 
-* **callback** `(error: Error | null, data: null) => void`
+* **callback** `(error: Error | null, data: null) => void` (optional) - If a `callback` is not provided, a `Promise` is returned
```

Comment:
```suggestion
* **callback** `(error: Error | null, data: null) => void` (optional)
```

Reply:
Duplicate info. You already mention this below.

Reply:
Do you want that description line removed for all of the callback arguments?